<launch>

<!-- <node pkg="learning_from_demonstration" name="marker_wrt_ee_publisher" 
type="marker_wrt_ee_publisher.py" output="screen"/>
  -->

<include file="$(find teleop_control)/launch/teleop_control.launch" >
    <arg name="part_to_publish" value="both" />
</include>

<include file="$(find aruco_ros)/launch/marker_publisher.launch">
    <!-- <arg name="ref_frame" value="xtion_rgb_optical_frame"/> -->
    <arg name="ref_frame" value="base_footprint"/>

</include>

<node pkg="learning_from_demonstration" name="trajectory_teaching" type="trajectory_teaching.py" output="screen">
</node>


</launch>